File: Python/PRAK604-2210817310017-Rohid Akmal.py
bintang = 0; pagar = 0
pesan1 = input(); pesan2 = input()
i1 = len(pesan1); i2 = len(pesan2)
if i1 != i2 :
    print("\nPanjang kalimat berbeda, pesan palsu\n")
else :
    for i in range(i2):
        if(pesan1[i] == pesan2[i]):
            if(pesan1[i] == ' '):
                print(" ", end=' ')
            else:
                print("*", end=' ')
                bintang += 1
        else:
            print("#", end=' ')
            pagar += 1
    print("\n* = ", bintang)
    print("# = ", pagar)
if(bintang >= pagar):
    print("Pesan Asli")
else:
    print("Pesan Palsu")

File: C/PRAK604-2210817310017-Rohid_Akmal.c
#include <stdio.h>
#include<string.h>

void main (){
    char pesan1[100], pesan2[100];
    int i1, i2, a, bintang = 0, pagar = 0;
    scanf("%[^\n]%*c", &pesan1);
    scanf("%[^\n]%*c", &pesan2);
    i1 = strlen(pesan1);
    i2 = strlen(pesan2);
    if(i1 != i2){
        printf("\nPanjang kalimat berbeda, pesan palsu\n");}
    else{
        for(a = 0; a < i1; a++){
            if(pesan1[a] == pesan2[a]){
                if(pesan1[a] == ' '){
                    printf(" ");}
                else{
                    printf("*");
                    bintang++;}}
            else{
                printf("#");
                pagar++;}}
        printf("\n\n* = %d", bintang);
        printf("\n# = %d", pagar);
        if(bintang >= pagar){
            printf("\nPesan asli");}
        else{
            printf("\nPesan Palsu\n");}
    }    
}